salt/states/boto_lambda.py

```diff
@@ -83,6 +83,7 @@ def __virtual__():
 def function_present(name, FunctionName, Runtime, Role, Handler, ZipFile=None, S3Bucket=None,
             S3Key=None, S3ObjectVersion=None,
             Description='', Timeout=3, MemorySize=128,
+            VpcConfig=None,
```

Comment:
The same thing applies here, too, about moving the new kwarg to end of the list.